mftool.sh

```
@@ -0,0 +1,58 @@
+if [ -z "$MF_FORK_URL" ]; then MF_FORK_URL=$(cat mf_fork_url.local); fi
+if [ -z "$MF_FORK_URL" ]; then echo "error: not set MF_FORK_URL"; exit 1; fi
+
+if [ -z "$TASK" ]; then echo "error: not set TASK"; exit 1; fi
+
+echo "MF_FORK_URL: $MF_FORK_URL"
+echo "TASK: $TASK"
+
+
+function use_fork() {
+    git submodule set-url muse $MF_FORK_URL
+    cd muse
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)